jobs/Scripts/utils.py

```diff
@@ -126,6 +126,10 @@ def driver_desktop():
 
     options = webdriver.ChromeOptions()
     options.binary_location = "C:\\Program Files\\AMD\\AMD RenderStudio\\AMD RenderStudio.exe"
+
+    if "scene_path" in case:
```

Comment:
Стоит добавить возможность указать пустой scene_path, чтобы студия оставалась на Home Screen
```suggestion
    if "scene_path" in case and case["scene_path"]:
```